src/vue-pdf-embed.vue

```diff
@@ -1,6 +1,13 @@
 <template>
-  <div>
-    <canvas v-for="pageNum in pageNums" :key="pageNum"></canvas>
+  <div v-if="documentId" :id="documentId">
+    <canvas
+      v-for="pageNum in pageNums"
+      :key="pageNum"
+      :id="`${documentId}-${pageNum}`"
+    />
+  </div>
+  <div v-else>
+    <canvas v-for="pageNum in pageNums" :key="pageNum" />
```

Comment:
Let's just override `id` for simplicity

```suggestion
  <div :id="id">
    <canvas
      v-for="pageNum in pageNums"
      :key="pageNum"
      :id="id && `${id}-${pageNum}`"
    />
```